tests/cassandra-stress/cassandra-stress-normal.yaml

```diff
@@ -25,7 +25,7 @@ spec:
       capabilities:
         add: ["IPC_LOCK"]
     command: ["/bin/sh"]
-    args: ["-c", "cassandra-stress 'user profile=/opt/cassandra-stress/normal_stress.yaml ops(insert=10,query_by_sub_id=8) duration=120m cl=one -node cassandra-demo-dc1.cassandra-demo -mode native cql3 user=bench password=monbench -rate threads=30 -pop seq=0..100M -tokenrange -graph file=/tmp/stress-normal.html' && echo END && while true ; do sleep 60; done"]
+    args: ["-c", "cassandra-stress 'user profile=/opt/cassandra-stress/normal_stress.yaml ops(insert=10,query_by_sub_id=8) duration=120m cl=one -node cassandra-demo -mode native cql3 user=bench password=monbench -rate threads=30 -pop seq=0..100M -tokenrange -graph file=/tmp/stress-normal.html' && echo END && while true ; do sleep 60; done"]
```

Comment:
don't you want to configure also the node name ? here cassandra-demo, and also login/password used to connect ?

Reply:
node name is already available https://github.com/Orange-OpenSource/cassandra-k8s-operator/pull/93/files#diff-b67911656ef5d18c4ae36cb6741b7965R462

for login/password, I think it's a good idea. I'll add that feature

Reply:
Just added 2 new options `USERNAME` and `PASSWORD`.